src/pages/community/components.mdx

```diff
@@ -13,8 +13,13 @@ _This content is accessible to IBMers only._
 
 <br />
 
-| Community components                                                           |                                                                                                                                       |
-| ------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------- |
-| [Cards](https://pages.github.ibm.com/cdai-design/pal/components/card/usage)    | _Accessible to IBMers only._ A flexible container that groups related blocks of content and information into one understandable unit. |
-| [Enhanced data table](https://pages.github.ibm.com/ibmcloud/pal/components/enhanced-data-table/code) | _Accessible to IBMers only._ Offers additional components that can be layered onto the default Carbon data table. Functionalities include row height settings, hide and show columns, and several filtering options.                           |
-| [Widget](https://pages.github.ibm.com/cdai-design/pal/components/widget/usage) | _Accessible to IBMers only._ Provides a summary of a platform functionality, status, or individual service.                           |
+| Community components                                                                                    |                                                                                                                                                                                         |
+| ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Cards](https://pages.github.ibm.com/cdai-design/pal/components/card/usage)                             | A flexible container that groups related blocks of content and information into one understandable unit.                                                                                |
```

Comment:
Should we use [snip](https://snip.innovate.ibm.com/) for the IBM-internal URLs?

Reply:
What is the benefit of Snip? Are we able to track who clicks it?

Reply:
It should give us basic stats of how many.

Though to integrate with our Fathom stats, could also do event tracking that way.

I think it's worth doing, I don't think it's a blocker to merging this PR.

Reply:
@connor-leech snip shortens links and helps us make sure internal IBM links aren't shared externally.

Reply:
@joshblack I'm down to move our links to Snip but I think it would be worthwhile to do so across our site. Going to merge this one with the links as is for now and open an issue for swapping links to Snip.